.github/actions/compile-models/requirements.txt

```diff
@@ -135,11 +135,11 @@ fsspec==2025.3.2 \
     # via
```

Comment:
colorama is removed here https://github.com/openvinotoolkit/npu_compiler/pull/76/files#diff-4ce361c39542bd48a9bf135d5ca98ec3f07221a7497e8f1bb79494b46d112763L111